### ISSUE TITLE
Menu lacks appropriate name, state, role, value/ keyboard focus

### ISSUE THREAD

**Issue by karlgroves** [jsbin/jsbin#1919]
Menu(s) identified as `<div class="menu">`

The menus on JSBin aren't accessible.  They should expose the necessary name, role, state, and value(s) appropriate for the menu and menuitem roles:
http://www.w3.org/TR/wai-aria/roles#menu
http://www.w3.org/TR/wai-aria/roles#menuitem

Additionally, they should have the appropriate key bindings and focus management.

I'd like to issue a PR against this, but unsure when I can get to it.
